src/main/java/com/example/bookshop/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package com.example.bookshop.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Column;
+import lombok.Data;
+import java.math.BigDecimal;
+
+@Entity
+@Data

```

Comment:
Lombok is a great library but be [attentive when using it with Entity](https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/) JFYI